scrabbler/scrabbler.py

```diff
@@ -103,12 +103,17 @@ def find_best_moves(self, rack, num=5):
             moves = across_moves + down_moves
 
         moves.sort(key=lambda move_: move_.score, reverse=True)
-        for move in moves[0:num]:
-            print(move)
+        # for move in moves[0:num]:
+        #     print(move)
+        return moves
 
     def show(self):
         """prints the board to terminal"""
         print(self.board)
+
+    def get_board(self):
```

Comment:
It looks like private fields are denoted by a leading `_` and public fields are not. I think we can directly access `board` in your driver class (i.e. `game.board`) and drop this getter.